### PR TITLE
ci: replace directory system log tables artifacts with tsv

### PR DESCRIPTION
**Pull request by azat**
Changelog:
- remove query_thread_log (query_log should be enough)
- for s3 storage there /var/lib/clickhouse/data/system/*_log cannot be
  used, replace them with plain TSVWithNamesAndTypes, also note, that
  after schema inference it is even pretty easy to use
- support replicated database env correctly

Changelog category (leave one):
- Not for changelog (changelog entry is not required)

### CODE REVIEW

**Comment by azat** [comment]
>for s3 storage there /var/lib/clickhouse/data/system/*_log cannot be
>used, replace them with plain TSVWithNamesAndTypes, also note, that
>after schema inference it is even pretty easy to use

Here is an example [1]:

```
tar: /var/lib/clickhouse/data/system/text_log: Cannot stat: No such file or directory
```

  [1]: https://s3.amazonaws.com/clickhouse-test-reports/32928/ddd5bebe555ce8feebcdd339e47fc45184c20dd1/stateless_tests__release__s3_storage__actions_/runlog.log

File: docker/test/stateless/run.sh [hunk 1]
```diff
@@ -131,8 +131,22 @@ clickhouse-client -q "system flush logs" ||:
...
+# NOTE:
+# - that due to tests with s3 storage we cannot use /var/lib/clickhouse/data
+#   directly
+# - even though ci auto-compress some files (but not *.tsv) it does this only
+#   for files >64MB, we want this files to be compressed explicitly
+for table in query_log zookeeper_log trace_log
+do
+    clickhouse-client -q "select * from system.$table format TSVWithNamesAndTypes" | pigz > /test_output/$table.tsv.gz &
+    if [[ -n "$USE_DATABASE_REPLICATED" ]] && [[ "$USE_DATABASE_REPLICATED" -eq 1 ]]; then
+        clickhouse-client --port 19000 -q "select * from system.$table format TSVWithNamesAndTypes" | pigz > /test_output/$table.replica.tsv.gz &
```
Comment (tavplubix) [inline docker/test/stateless/run.sh:146]:
```suggestion
        clickhouse-client --port 19000 -q "select * from system.$table format TSVWithNamesAndTypes" | pigz > /test_output/$table.1.tsv.gz &
                clickhouse-client --port 29000 -q "select * from system.$table format TSVWithNamesAndTypes" | pigz > /test_output/$table.2.tsv.gz &
```

**Review by tavplubix — APPROVED** [review]

File: docker/test/stateless/run.sh (hunk 1 above)
Comment (azat) [reply docker/test/stateless/run.sh:146]: I like more meaningful names, but OK, let's make it similar to other artifacts.

File: docker/test/stateless/run.sh [hunk 2]
```diff
@@ -131,8 +131,22 @@ clickhouse-client -q "system flush logs" ||:
...
+# NOTE:
+# - that due to tests with s3 storage we cannot use /var/lib/clickhouse/data
+#   directly
+# - even though ci auto-compress some files (but not *.tsv) it does this only
+#   for files >64MB, we want this files to be compressed explicitly
+for table in query_log zookeeper_log trace_log
+do
+    clickhouse-client -q "select * from system.$table format TSVWithNamesAndTypes" | pigz > /test_output/$table.1.tsv.gz &
+    if [[ -n "$USE_DATABASE_REPLICATED" ]] && [[ "$USE_DATABASE_REPLICATED" -eq 1 ]]; then
+        clickhouse-client --port 19000 -q "select * from system.$table format TSVWithNamesAndTypes" | pigz > /test_output/$table.2.tsv.gz &
```
Comment (tavplubix) [inline docker/test/stateless/run.sh:146]:
```suggestion
        clickhouse-client --port 19000 -q "select * from system.$table format TSVWithNamesAndTypes" | pigz > /test_output/$table.1.tsv.gz &
        clickhouse-client --port 29000 -q "select * from system.$table format TSVWithNamesAndTypes" | pigz > /test_output/$table.2.tsv.gz &
```

The previous comment was not about naming, but about number of files :)
There are 3 servers (listening ports 9000, 19000 and 29000) and we should save `zookeeper_log` from all 3 servers.
We can name files like `$table.replica1.tsv.gz` and `$table.replica2.tsv.gz` or `$table.replica1.tsv.gz` and `$table.shard1.tsv.gz` or `$table.s1r2.tsv.gz` and `$table.s2r1.tsv.gz`, it does not matter.

Comment (azat) [reply docker/test/stateless/run.sh]: Oh, completely forget about this, thanks! Fixed.

**Comment by azat** [comment]
Artifacts looks good (checked zookeeper_log fo s3 storage, databasereplicated and regular)

**Pull request merged**